cmd/boot-aws/main.go

```diff
@@ -12,10 +10,71 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 
 	"github.com/osbuild/images/internal/cloud/awscloud"
 )
 
+// exitCheck can be deferred from the top of command functions to exit with an
+// error code after any other defers are run in the same scope.
+func exitCheck(err error) {
+	if err != nil {
+		fmt.Fprintf(os.Stderr, err.Error())
+		os.Exit(1)
+	}
+}
+
+// checkFlags checks that all the required flags are set and returns error if
+// one is not.
+func checkFlags(flags *pflag.FlagSet) error {
```

Comment:
Use https://pkg.go.dev/github.com/spf13/cobra#MarkFlagRequired instead of a custom function?

Reply:
Done!